pytorch_lightning/callbacks/lr_logger.py

```diff
@@ -80,7 +80,8 @@ def on_batch_start(self, trainer, pl_module):
             if trainer.logger and latest_stat:
                 trainer.logger.log_metrics(latest_stat, step=trainer.global_step)
         else:
-            pass
+            raise MisconfigurationException(
+                'logging_interval should be `step` or `epoch` or None.')
```

Comment:
@aiyolo I would prefer this check in `__init__` itself, so that one should know this while creating the callback rather than after doing `.fit` and remove the else part completely :)

Reply:
thanks for your suggestion